csunplugged/templates/base.html

```diff
@@ -6,13 +6,20 @@
 
 {% get_language_info for LANGUAGE_CODE as current_language %}
 
+
 <!DOCTYPE html>
 <html lang="{{ LANGUAGE_CODE }}" {% if current_language.bidi %}dir="rtl"{% endif %}>
   <head>
     <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
+    {% if LANGUAGE_CODE == "xx-lr" or LANGUAGE_CODE == "yy-rl" %}
```

Comment:
Do we want to display these in the language picker to make it easier for translators? Would be easy to do.

Reply:
They would be automatically, if the language picker was implemented. Perhaps instead of removing it completely, we should add it to develop, but wrapped in an `if` block that checks if the length of `LANGUAGES` is greater than 1?